remote_ikernel/manage.py

```diff
@@ -191,66 +193,66 @@ def add_kernel(
     user.
     """
     kernel_name = []
-    display_name = []
+    display_name = cluster_name + "_" + cluster_uuid[:5]
```

Comment:
```suggestion
    display_name =  f"{cluster_name} (ID: {cluster_uuid})"
```
I assume it'll automatically get truncated if it's too long?

Reply:
I think so.